src/config/connection.ts

```diff
@@ -0,0 +1,14 @@
+import { Sequelize } from 'sequelize';
+require('dotenv').config();
+
+let dbUrl;
+
+if (process.env.NODE_ENV === 'test') {
```

Comment:
convert to switch or create an object and handle database in case of production 
```js
const {NODE_ENV, DATABASE_URL} = process.env;
const dbUrls = {
'production': DATABASE_URL, // ...etc
}
let dbURl = dbUrls[NODE_ENV]
```